mne/chpi.py

```diff
@@ -295,14 +295,51 @@ def extract_chpi_locs_kit(raw, stim_channel='MISC 064', *, verbose=None):
 
 # ############################################################################
 # Estimate positions from data
+
 @verbose
-def _get_hpi_info(info, allow_empty=False, verbose=None):
-    """Get HPI information from raw."""
+def get_chpi_info(inst, on_missing='ignore', verbose=None):
+    """Retrieve cHPI information from the data.
+
+    Parameters
+    ----------
+    inst : instance of Info | Raw | Epochs | Evoked
+        The object from which to extract the HPI information.
+    on_missing : 'ignore' | 'raise'
+        What to do if no HPI information can be found. If ``'ignore'``
+        (default), all return values will be empty arrays or ``None``. If
+        ``'raise'``, an exception will be raised.
```

Comment:
Better / more DRY to use `on_missing_base` in `docs.py` (see how we do it for other `on_*` in other functions) and the helper `mne.utils. _on_missing`. I don't see a reason why `'warn'` shouldn't be supported here.